From abd71c2e9a04f9e2747a96d916917176e6637d26 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 17 Nov 2022 09:52:19 +0000
Subject: [PATCH 2/2] gnu: python-sniffio: Update to 1.3.0

* gnu/packages/python-xyz.scm (python-sniffio): Update to 1.3.0.
  [build-system]: Use pyproject-build-system to simplify package.
---
 gnu/packages/python-xyz.scm | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/gnu/packages/python-xyz.scm b/gnu/packages/python-xyz.scm
index 8de175dd1a..d3fd727042 100644
--- a/gnu/packages/python-xyz.scm
+++ b/gnu/packages/python-xyz.scm
@@ -12342,20 +12342,14 @@ (define-public python-tables
 (define-public python-sniffio
   (package
     (name "python-sniffio")
-    (version "1.2.0")
+    (version "1.3.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "sniffio" version))
        (sha256
-        (base32 "1pnkvi6wvn4qd37y69n1ls6n8l63gfmn3pvb1jb50gqxxkn6wrn4"))))
-    (build-system python-build-system)
-    (arguments
-     `(#:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda _
-             (invoke "pytest" "-vv"))))))
+        (base32 "00cibg2madxg6j912qwpwkgggn1km8mg5dsrj8wga56kwp2ha0z6"))))
+    (build-system pyproject-build-system)
     (native-inputs
      (list python-curio python-pytest python-pytest-cov))
     (home-page "https://github.com/python-trio/sniffio")
-- 
2.38.0

